asaps/workflows.py

```diff
@@ -0,0 +1,31 @@
+import csv
+
+from asaps import models, records
+
+
+def export_metadata(client, resource, field, repo_id):
+    """Export ArchivesSpace metadata as a CSV."""
```

Comment:
I will be using this idea elsewhere but I'm going to keep this as is because the field specified is added as the `file_identifier` which is the match point for the file names, not the field's name in ArchivesSpace. The ArchivesSpace field that serves as the match point is going to vary based on how files were named over the years (efforts are underway to make it consistent going forward)

Reply:
That makes sense, however if that's the case then this is a very specific function whose purpose is to retrieve the file identifier, and I would rename it and document it as such.

Reply:
Great point, will do!